src/main/java/frc/robot/Controls.java

```diff
@@ -372,6 +380,33 @@ private void configureSuperStructureBindings() {
                   .withName("Automatic Intake"));
     }
 
+    if (sensors.armSensor != null) {
+      sensors
+          .armSensor
+          .inClaw()
+          .and(RobotModeTriggers.teleop())
+          .onTrue(
+              Commands.runOnce(
+                      () -> {
+                        switch (intakeMode) {
+                          case CORAL -> soloScoringMode = SoloScoringMode.CORAL_IN_CLAW;
+                          case ALGAE -> soloScoringMode = SoloScoringMode.ALGAE_IN_CLAW;
+                        }
+                      })
+                  .withName("Set solo scoring mode"));
+
+      sensors
+          .armSensor
+          .inClaw()
+          .and(RobotModeTriggers.teleop())
+          .onFalse(
```

Comment:
Can this .onFalse() move to after the .onTrue() function that starts on line 388?  Do we need to dereference through sensors again?